### PR TITLE
Fix invalid Source0

### PR DESCRIPTION
**Pull request by vojtechsokol**

### CODE REVIEW

File: packaging/leapp-repository.spec [hunk 1]
```diff
@@ -21,7 +22,7 @@ Summary:        Repositories for leapp
 
 License:        ASL 2.0
 URL:            https://oamg.github.io/leapp/
-Source0:        https://github.com/oamg/leapp-repository/archive/leapp-repository-%{version}.tar.gz
+Source0:        https://github.com/oamg/%{name}/archive/%{gittag}/%{name}.tar.gz
```
[the remark this answers is not in the document]
Comment (vojtechsokol) [reply packaging/leapp-repository.spec]: After macro expansion the Source0 without `%{gittag}` (and with `%{version}` which I forgot) looks like this:
```
https://github.com/oamg/leapp-repository/archive/leapp-repository-0.7.0.tar.gz
```
but that returns 404. The gittag field is mandatory for valid URL.

[the remark this answers is not in the document]
Comment (bocekm) [reply packaging/leapp-repository.spec]: The working link is: https://github.com/oamg/leapp-repository/archive/v0.7.0.tar.gz

[the remark this answers is not in the document]
Comment (pirat89) [reply packaging/leapp-repository.spec]: @bocekm  you need to change name as well. so the link above

[the remark this answers is not in the document]
Comment (vojtechsokol) [reply packaging/leapp-repository.spec]: Yes, the basename of the source needs to be in the form of `%{name}-%{version}.tar.gz` to satisfy rpmbuild.

**Pull request merged**